corehq/apps/app_manager/templates/app_manager/v2/partials/custom_icon.html

```diff
@@ -0,0 +1,31 @@
+{% load xforms_extras %}
+{% load i18n %}
+
+<div>
+    <div class="form-group container-fluid">
+        <label class="control-label col-sm-2">{% trans "Custom Icon" %}</label>
+        <div class="col-sm-10 commcare-feature" data-since-version="1.3">
+            <label>
+                <span>
+                    {% trans "Form" %}
+                </span>
+                <input class="form-control" name="custom_icon_form" id="custom_icon_form"
```

Comment:
I'd like to push back on that - this is something that'll be visible to all users, and having a text box where you need to type in a specific word, but you don't even know what the options are, is pretty low usability. Should this be behind a feature flag instead?